tests/test_common.py

```diff
@@ -8,7 +8,7 @@
 CONTRACT = "contract"
 OPERATION = "invoke"
 NETWORK = "goerli"
-ARGS = [1, 2, 3]
+ARGS = ["1", "2", "3"]
```

Comment:
I'm wondering if we should keep the `ARGS` as ints and assert they come out as strings in the subprocess. Maybe it's not necessary fi we isolate and test `stringify`

Reply:
i prefer the shortest path at this point 😅